packages/valory/skills/abstract_round_abci/tests/test_models.py

```diff
@@ -165,8 +166,8 @@ def test_get_spec(
         actual_specs = {
             "url": "http://dummy",
             "method": "GET",
-            "headers": [("Dummy-Header", "dummy_value")],
-            "parameters": [("Dummy-Param", "dummy_param")],
+            "headers": {"Dummy-Header": "dummy_value"},
+            "parameters": {"Dummy-Param": "dummy_param"},
```

Comment:
@angrybayblade originally introduced the list type I believe. Please check if you're happy